<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eightPuzzle</class>
 <widget class="QWidget" name="eightPuzzle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eightPuzzle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>方正小标宋简体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>初始状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>方正小标宋简体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>终止状态</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="intial" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="i_1">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/one.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="i_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/two.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="i_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/three.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="i_4">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/four.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="i_5">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/five.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="i_6">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/six.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="i_7">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/seven.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="i_8">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/eight.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="i_0">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/zero.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="end" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QLabel" name="e_8">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/eight.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="e_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/two.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="e_1">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/one.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="e_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/three.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="e_5">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/five.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="e_7">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/seven.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="e_4">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/four.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="e_6">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/six.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="image.qrc">:/image/zero.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QListView" name="View"/>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="autoPutton">
        <property name="text">
         <string>自动演示</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="init">
        <property name="text">
         <string>设置初始状态</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="printImage">
           <property name="text">
            <string>打印搜索过程图形版</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="printTree">
        <property name="text">
         <string>打印搜索过程</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>选择启发函数</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="h1">
           <property name="text">
            <string>h1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="h2">
           <property name="text">
            <string>h2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
